modules/argocd-installing-cli.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-argocd.adoc
+
+[id="argocd-installing-cli_{context}"]
+= Installing the ArgoCD command-line tool
+
+To manage your ArgoCD instance from your local machine, install the ArgoCD command-line tool.
+
+.Procedure
+
+. Navigate to the link:https://github.com/argoproj/argo-cd/releases/latest[Releases] page in the ArgoCD project repository and note the number of the latest release, such as `v1.4.0`.
+
+. Download the ArgoCD command-line tool to a directory that is defined as `$PATH`:
++
+----
+# curl -sSL -o /usr/local/bin/argocd https://github.com/argoproj/argo-cd/releases/download/<version>/argocd-linux-amd64 <1>
```

Comment:
@dgoodwin should this refer to the specific version of the CLI that we have verified? Or are they backward compatible?

Reply:
@kalexand-rh could this refer to the ArgoCD installation docs for CLI?
https://argoproj.github.io/argo-cd/cli_installation/

If not, we should also include links for downloading on Mac

Reply:
In my experience they have not been backward compatible.

@abutcher what versions have you been testing with? I've been on 1.3.x, I don't think I used 1.4.x in my testing.

Reply:
I've only been using 1.3.x thus far.

Reply:
I have been testing 1.4.1 and 1.4.2 version. 1.3.x maybe have problems in configuring multiple clusters